### PR TITLE
Readme cleanup

### PR DESCRIPTION
**Pull request by muuki88**
The current readme bothered me for quite some time as information is stale, no easy starting point and not enough pointers to our documentation. I tried to improve this.

@kardapoltsev @fiadliel WDYT?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,26 +26,32 @@ join the discussion in [#498](https://github.com/sbt/sbt-native-packager/issues/
...
   * Universal `zip`,`tar.gz`, `xz` archives
   * `deb` and `rpm` packages for Debian/RHEL based systems
   * `dmg` for OSX
   * `msi` for Windows
   * `docker` images
 * Provide archetypes for common use cases
- * Java application with startscripts for linux/osx/windows
- * Java server additional autostart configurations
+ * [Java application] with startscripts for linux/osx/windows
+ * [Java server application][] with additional autostart configurations
```
Comment (fiadliel) [inline README.md]: I'm not sure how this link format is supposed to work, but it seems to go to the same link as "Java application" above.

File: README.md [hunk 2]
```diff
@@ -31,26 +26,32 @@ join the discussion in [#498](https://github.com/sbt/sbt-native-packager/issues/
...
+   * Systemd
+   * Systemv
+   * Upstart
+* Java8 [jdkpackager][] wrapper
+* Optional JDeb integration for crossplattform debian builds
```
Comment (fiadliel) [inline README.md]: cross-platform

Comment (fiadliel) [inline README.md]: debian -> Debian

File: README.md [hunk 3]
```diff
@@ -109,7 +99,10 @@ There's a complete "getting started" guide and more detailed topics available at
 
 Please feel free to [contribute documentation](https://github.com/sbt/sbt-native-packager/tree/master/src/sphinx), or raise issues where you feel it may be lacking.
 
-## Related SBT Plugins
+## Related SBT Plugins ##
+
+These are a list of plugins that either use sbt-native-packager, provide additional features
+or provide a richer API for a single packaging format.
 
 - [sbt-heroku](https://github.com/heroku/sbt-heroku)
```
Comment (fiadliel) [inline README.md]: While the README is being updated, how about putting this list in alphabetical order?

File: README.md [hunk 4]
```diff
@@ -121,3 +114,10 @@ Please feel free to [contribute documentation](https://github.com/sbt/sbt-native
 - [sbt-newrelic](https://github.com/gilt/sbt-newrelic)
 - [sbt-aether](https://github.com/arktekk/sbt-aether-deploy)
 - [sbt-package-courier](https://github.com/alkersan/sbt-package-courier)
+
+
+[formats]: http://www.scala-sbt.org/sbt-native-packager/gettingstarted.html#packaging-formats
+[Java application]: http://www.scala-sbt.org/sbt-native-packager/archetypes/java_app/index.html
+[Java server application]: http://www.scala-sbt.org/sbt-native-packager/archetypes/java_app/index.html
```
Comment (fiadliel) [inline README.md]: Ah, here's where the link is. Change this to point to the server archetype.

File: README.md [hunk 5]
```diff
@@ -1,24 +1,19 @@
...
+[ ![Download](https://api.bintray.com/packages/sbt/sbt-plugin-releases/sbt-native-packager/images/download.svg) ](https://bintray.com/sbt/sbt-plugin-releases/sbt-native-packager/_latestVersion)
+[![Join the chat at https://gitter.im/sbt/sbt-native-packager](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/sbt/sbt-native-packager?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Issue Stats](http://www.issuestats.com/github/sbt/sbt-native-packager/badge/pr?style=flat)](http://www.issuestats.com/github/sbt/sbt-native-packager)
+[![Issue Stats](http://www.issuestats.com/github/sbt/sbt-native-packager/badge/issue?style=flat)](http://www.issuestats.com/github/sbt/sbt-native-packager)
 
+## Goal ##
 
-This is a work in progress project.  The goal is to be able to bundle up Scala software built with SBT for native packaging systems, like deb, rpm, homebrew, msi.
+SBT native packager lets you build application packages in native formats. It offers
+different archetypes for common configurations, such as simple java apps or server applications.
```
Comment (fiadliel) [inline README.md]: java -> Java

**Comment by fiadliel** [comment]
Some small comments, but looks great!

File: README.md (hunk 3 above)
Comment (muuki88) [inline README.md]: Good point. I will update this.

File: README.md (hunk 1 above)
Comment (muuki88) [inline README.md]: There are two ways of defining links in markdown the `[Description][Link Name]` is used by default at Stackoverflow.
The inline variant `[Description](link)` clutters the text if not standalone.

**Pull request merged**